generator/idl_parser/idls.ts

```diff
@@ -29,19 +29,43 @@ export interface InterfaceMember extends TypedIdentifier {
 export const enum DefinitionType {
   Interface,
   Enum,
+  ExtendedAttribute,
```

Comment:
i think ExtendedAttribute is not same level with Interface or Enum.
(refer to around this lines https://github.com/lunchclass/bacardi/blob/master/generator/idl_parser/idls.ts#L107)
so i think it is better to remove here.
and i am considering to remove DefinitionType enums.
because we can distinguish types from object using |instanceof| keyword without type attributes.

Reply:
Okay I try to remove and check other solution .